packages/preview-middleware-client/test/unit/adp/controllers/AddFragment.controller.test.ts

```diff
@@ -517,9 +519,24 @@ describe('AddFragment', () => {
             });
             rtaMock.getFlexSettings.mockReturnValue({ projectId: 'adp.app' });
 
+            const overlays = {
+                getId: jest.fn().mockReturnValue('some-id')
+            };
+
+            jest.spyOn(sap.ui, 'getCore').mockReturnValue({
```

Comment:
You should be able to use mock core `import {  sapCoreMock } from 'mock/window';`.

Reply:
adapted